README.md

```diff
@@ -50,35 +50,36 @@ pip install git+https://github.com/gridsingularity/d3a-api-client.git
 
 ## How to use the Client
 
-### Interacting with the CLI
-You can run one of the setup scripts residing in the `d3a_api_client/setups` folder, or you can 
-set the path for your scripts directory as list in the `d3a_api_client.constants` .
-In order to run a module you can use the following command
-```
-d3a-api-client run --setup {module_name}
-```
+### Interacting via CLI
+In order to get help, please run:
+```
+d3a-api-client run --help
+```
+Following is the more in depth explanation of individual flags supported,
+- base-setup-path --> Sets the base path where you client script resides else `d3a_api_client/setups` is taken as default for your client-scripts. User could provided both an absolute and relative file path.
+- setup --> Name of your api-client module/script.
+- username --> Username of agent authorized to communicate with respective collaboration/CN.
+- password --> Password of respective agent
+- domain-name --> D3A domain name
+- web-socket --> D3A websocket URI
+- simulation-config-path --> Path of JSON format file that contains agent's point of interest's collaboration/CN's info (It could downloaded from the registry page of UI) 
```

Comment:
I'm not sure I understood exactly what you mean here by "agent's point of interest". This file contains the information regarding the collaboration/CN, correct?

Reply:
`agent's point of interest` --> each users requesting it would only getting its own approved areas and selected aggregator info. May be you can propose better wording here

Reply:
Ah, I see. Not easy...what about:
```suggestion
- `--simulation-config-path`: path to the JSON file that contains the user's collaboration/CN info (it can be downloaded from the registry page of the D3A website)
```
I think there's no need to be more specific, since you're already saying that this file will only contain information related to the user.